packages/frint-cli/commands/new.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-use-before-define */
+
+const mkdirp = require('mkdirp');
+const request = require('request');
+const tar = require('tar');
+
+const createApp = require('frint').createApp;
+
+const DEFAULT_ORGANIZATION = 'Travix-International';
+const DEFAULT_REPOSITORY = 'frint';
+const DEFAULT_BRANCH = 'master';
+const DEFAULT_EXAMPLES_DIR = 'examples';
+const DEFAULT_EXAMPLE = 'counter';
+const DESCRIPTION_TEXT = `
+Usage:
+
+  $ frint new
+  $ frint new <directory>
+  $ frint new <directory> --example <example>
+
+Example:
+
+  $ frint new myapp --example kitchensink
+  $ frint new myapp --example frint-vue/tree/master/examples/basic
+
+You can find a list of all available official examples here:
+https://github.com/Travix-International/frint/tree/master/examples
+`.trim();
+const INVALID_EXAMPLE_ARG_TEXT = `
+Invalid <example> value. Must be in one of the following formats:
+
+  * <name>
+  * <organization>/<repository>/tree/<branch>/**
+`.trim();
+const COMPLETION_TEXT = `
+Done!
+
+Please run these two commands to start your application:
+
```

Comment:
depending on whether `<directory>` argument was given or not, we can also add an addition `cd` command.

Two scenarios:

1: with directory:

```
$ frint new myapp

Done!

Please run:

$ cd myapp
$ npm install
$ npm start
```

2: without directory

```
$ frint new

Done!

$ npm install
$ npm start
```

Sorry for nitpicking. Just trying to come up with as many use cases as possible :)

Reply:
> Sorry for nitpicking. Just trying to come up with as many use cases as possible :)

No worries, good feedback is good!

I was thinking about it actually or alternatively adjusting the wording to specify to run the following commands in the *app folder*.